src/angular-resizable.js

```diff
@@ -1,5 +1,5 @@
 angular.module('angularResizable', [])
-    .directive('resizable', function() {
+    .directive('resizable', function($parse) {

```

Comment:
Besides, not adding the dependency as string also makes it break if using strict mode.